1.train_models/log_reg_plates_3_3p_5_cp_norm_data/nbconverted/plate_3_3p_5_train_log_reg_scv.py

```diff
@@ -321,45 +324,37 @@ def process_plates(_df):
 logreg.fit(X, y)
 
 
-# # Save models and model data
-
-# ## Save model
+# ## Store training data
 
 # In[12]:
 
 
-data_suf = "log_reg_cp_fs_data_plate_5"
+eval_data["plate"].extend(restdf["Metadata_Plate"].tolist())
```

Comment:
Double checking: would `eval_data` at this point still include the prior data which was extended to it above? If so, does this impact any analysis which would be performed on it?

Reply:
It would, yes, this was intentional